html/auth0/verify_email.html

```diff
@@ -40,9 +40,8 @@
               <p><a href="{{ url }}">Verify my email address</a></p>
 
               <p>
-                If you are having any issues with your account, please don’t hesitate to contact us <a href="{{ support_url }}">here.</a>
+                This link will expire after 24 hours. If you're link is expired or you are having any other issues with your account, please don’t hesitate to contact us <a href="{{ support_url }}">here.</a>
```

Comment:
Just a spelling mistake in here. "If you're link is expired..." should be "If your link is expired..."